docs/notebooks/des-gaia.ipynb

```diff
@@ -0,0 +1,660 @@
+{
```

Comment:
<p>&nbsp;&nbsp;# sort columns are optional and works only if few objects are very close to each other</p><p> </p><p>This is confusing. Why is it optional, and why does it work only when few objects are very close to each other.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I think I could just remove this option...</p><p> </p>

Reply:
<p>I removed the comment =)</p>